Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -0,0 +1,266 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #toolbar button { display: block; }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+    <select data-bind="options: tilesets, optionsText: 'name', value: selectedTileset, optionsCaption: 'Choose a Tileset...'"></select>
+    <div>
+        <input type="checkbox" data-bind="checked: annotate" /> Annotate
+        <input type="checkbox" data-bind="checked: shadows" /> Shadows
+    </div>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    shadows : true
+});
+viewer.extend(Cesium.viewerCesium3DTilesInspectorMixin);
+var inspectorViewModel = viewer.cesium3DTilesInspector.viewModel;
+
+var scene = viewer.scene;
+scene.pickTranslucentDepth = true;
+
+var annotations = scene.primitives.add(new Cesium.LabelCollection());
+
+var tileset;
+
+var viewModel = {
+    tilesets: [
+        {
+            name: 'Tileset',
+            url: '../../../Specs/Data/Cesium3DTiles/Tilesets/Tileset/'
+        }, {
+            name: 'Translucent',
+            url: '../../../Specs/Data/Cesium3DTiles/Batched/BatchedTranslucent/'
+        }, {
+            name: 'Translucent/Opaque',
```

Comment:
Yeah I'm aware of that, it's the last bullet point in the todo. It's a problem with the gltf, not Cesium.